lib/utilities.js

```diff
@@ -0,0 +1,13 @@
+import Usbmux from './usbmux/usbmux';
+import _ from 'lodash';
+
+
+async function getConnectedDevices (socket) {
+  const usbmux = new Usbmux(socket);
+  const devices = await usbmux.listDevices();
+  usbmux.close();
```

Comment:
shouldn't it be in `finally`?

Reply:
is it ok from performance perspective to create a new usbmux instance for each call @umutuzgur ?

Reply:
My vision would be that once we were using this for more than just this, the driver would have have a copy that could be used. But in order to be immediately useful, this is the only thing we need.

And it will be orders of magnitude faster than forking a new `idevice_id` process to get the information.

Reply:
It is okay to create and close the usbmux clients in this magnitude. We should just not make the APIs too simple and hit issues in the future for mishandling of resources